modules.js

```diff
@@ -6,183 +6,173 @@ const { Client } = require("@notionhq/client");
-                        .map((x) => ({
-                            name: x.replace(",", ""),
-                        })),
-                },
-
-                Description: {
-                    rich_text: [
-                        {
-                            text: {
-                                content:
-                                    (book.volumeInfo.description || "").length <
-                                    500
-                                        ? book.volumeInfo.description || ""
-                                        : book.volumeInfo.description.substring(
-                                              0,
-                                              500
-                                          ) + "...",
-                            },
-                        },
-                    ],
-                },
-
-                Link: {
-                    url: book.volumeInfo.previewLink,
-                },
+              type: "text",
+              text: {
+                content:
+                  book.volumeInfo.title ||
+                  i.properties.Title.title[0].plain_text.replace(";", ""),
+              },
             },
-
-            icon: {
-                emoji: "📖",
+          ],
+        },
+
+        "Author(s)": {
+          multi_select: book.volumeInfo.authors
+            .filter((x) => x)
+            .map((x) => ({
+              name: x.replace(",", ""),
+            })),
+        },
+
+        "Genre(s)": {
+          multi_select: (book.volumeInfo.categories || [])
+            .filter((x) => x)
+            .map((x) => ({
+              name: x.replace(",", ""),
+            })),
+        },
+
+        Description: {
+          rich_text: [
+            {
+              text: {
+                content:
+                  (book.volumeInfo.description || "").length < 500
+                    ? book.volumeInfo.description || ""
+                    : book.volumeInfo.description.substring(0, 500) + "...",
+              },
             },
-        };
-
-        if (book.volumeInfo.imageLinks) {
-            updateOptions.icon = {
-                external: {
-                    url: `https://readinglist-cover-proxy.srg.id.au/${book.id}.jpg`,
-                },
-            };
-
-            updateOptions.cover = {
-                external: {
-                    url: `https://readinglist-cover-proxy.srg.id.au/${book.id}.jpg`,
-                },
-            };
-        }
-
-        if (book.volumeInfo.averageRating) {
-            updateOptions.properties["Rating"] = {
-                number: book.volumeInfo.averageRating,
-            };
-        }
-
-        if (book.volumeInfo.pageCount) {
-            updateOptions.properties["Pages"] = {
-                number: book.volumeInfo.pageCount,
-            };
-        }
-
-        try {
+          ],
+        },
+
+        Link: {
+          url: `https://openlibrary.org/isbn/${isbn}`,
```

Comment:
Open Library asks for a link back if using their [covers API](https://openlibrary.org/dev/docs/api/covers), so this seemed like a harmless change for their service.